### PR TITLE
Feat reptile page (hero section)

### PR DESCRIPTION
**Pull request by mvic89**
Created HTML elements with JS.
Added CSS & responsivness to the hero section.

### CODE REVIEW

File: scripts/reptiles.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+let mainDiv = document.querySelector('main');
+
+let containerDiv = document.createElement('div');
+containerDiv.classList.add('container');
+mainDiv.appendChild(containerDiv);
+
+let imageElement = document.createElement('img');
```
Comment (Endorze) [inline scripts/reptiles.js]: Any element where you're fetching/creating something using document.querySelector/getElementById etc, then it should be created with Const as you will never modify this value.

The use of let is used for when you're changing something, such as a variable turning true/false, or any other value that should change when its used.

Comment (Endorze) [reply scripts/reptiles.js]: Also this is a lot of code, maybe consider making this a function?

File: styles/reptiles.css [hunk 2]
```diff
@@ -0,0 +1,122 @@
+
+.main-container {
+    min-height: 100vh;
+    width: 100%;
+    overflow: hidden;
+    display: flex;
+
+    min-height: 100vh;
```
Comment (Endorze) [inline styles/reptiles.css]: You have 2x of min-height: 100vh, remove one.

File: scripts/reptiles.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
+let mainDiv = document.querySelector('main');
```
Comment (maysunktong) [inline scripts/reptiles.js]: you can change 'let' to 'const' instead if you do not plan to reassign the variable. :)

File: styles/reptiles.css [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+    .hero-text h1 {
+        font-size: 2em;
+    }
+
+    .hero-text p {
+        font-size: 1rem;
+    }
+}
+
+@media screen and (max-width: 660px) {
```
Comment (Endorze) [inline styles/reptiles.css]: We're using 1024px, 768px on the assignment, could you consider changing it to match unless it breaks your content?

**Review by Endorze — APPROVED** [review]
Look at my comments, otherwise everything looks good 😄

**Review by maysunktong — APPROVED** [review]

**Pull request merged**